message/message.go

```diff
@@ -1,20 +1,19 @@
 package message
 
-// Message represents a message flowing through the wire. For example, a Kafka message.
-type Message struct {
-	Context Context  `json:"context"`
-	Key     []byte   `json:"key,omitempty"`
-	Value   []byte   `json:"value,omitempty"`
-	Headers []Header `json:"headers,omitempty"`
-}
+import (
+	"github.com/ThreeDotsLabs/watermill"
+	watermillmessage "github.com/ThreeDotsLabs/watermill/message"
+)
 
-// Context contains information about the context that surrounds a message.
-type Context struct {
-	Channel string `json:"channel"`
-}
+const (
+	// MetadataChannel is the key used for storing the Channel in the message Metadata.
+	MetadataChannel = "_asyncapi_eg_channel"
```

Comment:
`_eg_`? What is this short for?

Reply:
The idea is to give a unique-ish key here. `eg` stands for **e**vent **g**ateway.

Reply:
At first I thought about the `equivalent` (but where is `q` 😅 ). Ok, thanks for clarification. Maybe add to the comment info about this `eg`?

Reply:
Comment added!